### ISSUE TITLE
ERROR: Cannot take screenshot: screen grab failed

### ISSUE THREAD

**Issue by xme** [kevoreilly/CAPEv2#191]
## This is opensource and you getting __free__ support so be friendly!

# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository (for multi-repository projects)

# Expected Behavior

Installed new guest with config identical to others

Screenshot module is not working

# What is the current behavior?

2020-05-01 11:15:36,468 [modules.auxiliary.screenshots] ERROR: Cannot take screenshot: screen grab failed

## Context

Guest OS: Win7 x64
Python 3.8.2
Pillow 6.2.2

agent.py runs with high privileges

When I try manually, it works:

from PIL import ImageGrab
i = ImageGrab.grab()
i.savet("test.png")

I presume it's a permission issue but how to check deeper?

**Comment by doomedraven** [comment]
`agent.py runs with high privileges` doesnt' say to much, cmd with admin isn't the same as start on boot for example

tell us how you started agent/persistence? so we can dig deeper

**Comment by doomedraven** [comment]
any update?

**Comment by xme** [comment]
Sorry, I was busy for the last days... Yes, it's fixed, I found the issue.
I'm launching agent.py at startup via a scheduled task. The mistake I made: I executed the task at boot time even if the user was not yet logged in. You've to schedule the task when a user logs in. Apparently, it does not find the screen in the first case.
Tx!

**Comment by mashalzainab** [comment]
Hello, I have the same error and am also running the agent by scheduling the task on user logon. 
But cannot click screenshots.

**Comment by doomedraven** [comment]
Provide more details, what hypervisor? If KVM did you enable it in aux config? If not KVM, did you install pillow? What analysis.log says about screenshots?

**Comment by mashalzainab** [comment]
Thank you for the response, lowering the UAC level solved the problem for me just a few minutes ago. 
I was running Windows 7 on virtualbox with pillow 6.2.2 version and python 2.7.18. 
The analysis.log had the same error as this thread: Cannot take screenshot: screen grab failed. 
I tried playing with versions, the way I was scheduling the agent and its settings etc. and finally lowering the UAC level to Never Notify fixed it for me. Found it from https://github.com/cuckoosandbox/cuckoo/issues/1388 this issue.
